src/react-apps/ux-editor/src/components/config/ConditionalRenderingComponent.tsx

```diff
@@ -175,10 +177,48 @@ class ConditionalRendering extends React.Component<IConditionalRenderingComponen
     this.props.deleteConnection(this.props.connectionId);
   }
 
+  public renderConditionalRenderingTargetComponentOption = (id: string): JSX.Element => {
+    return (
+      <option key={id} value={id}>
+        {this.props.formLayoutComponents[id].title + ' (id=' + id + ')'}
+      </option>
+    );
+  }
+
+  public renderCondtionalRenderingTargetContainerOptions = (id: string, baseContainer?: boolean): JSX.Element[] => {
+    let options: JSX.Element[] = [];
+    if (!baseContainer) {
+      options.push(
+        <option key={id} value={id}>
+          {'Container (id=' + id + ')'}
+        </option>
+      );
+    }
+    this.props.order[id].forEach(key => {
+      if (this.props.formLayoutComponents[key]) {
+        let option = this.renderConditionalRenderingTargetComponentOption(key);
+        options.push(option);
+      }
+      else {
+        // A container can have compnents and sub-containers
+        let containerOptions = this.renderCondtionalRenderingTargetContainerOptions(key);
+        containerOptions.forEach(option => {
+          options.push(option);
+        })
+      }
+    })
+    return options;
+  }
+
+  public renderCondtionalRenderingTargetOptions = (): JSX.Element[] => {
+    let baseContainerKey = Object.keys(this.props.order)[0];
```

Comment:
use `const` when the value is not going to be changed.

Reply:
Fixed.